chain/get_ancestors.go

```diff
@@ -99,6 +83,7 @@ func CollectTipSetsOfHeightAtLeast(ctx context.Context, iterator *TipsetIterator
 		}
 		ret = append(ret, iterator.Value())
```

Comment:
Blocking: Hey @deaswang in order to close out #3027 this function needs a behavior change.  It looks like the original issue wasn't clear about this.  I've updated the issue to [explain this in better detail](https://github.com/filecoin-project/go-filecoin/issues/3027#issuecomment-523106427).  Please follow up with questions here or there if it is still unclear.